smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -226,13 +227,16 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
         pow += getValidatorPower(validator.signer);
       }
 
-      // Signatures must be sorted on the sifchain side, so we just
-      // need to make sure that the next validator in the array
+      // Signatures must be sorted on the relayer side, so we just
+      // need to make sure that the next witness in the array
```

Comment:
What is the actual sorting key? (i.e. `validator.signer`)

Reply:
sorting key is signer, cosmosBridge need check if the duplicated signer in the signature list. sorted singer make it efficient to compare, gas saving.